config/locales/en.yml

```diff
@@ -152,6 +152,48 @@ en:
           friend at %{invitation_page}.
           Cheers,
           %{archive_name}
+    claim_notification:
+      subject: "[%{app_name}] Works uploaded"
+      open_doors: "Open Doors"
+      open_doors_faq: "FAQ page"
+      contact_open_doors: "contact Open Doors"
+      ao3: "Archive of Our Own"
+      ao3_news: "AO3 News"
+      tutorials_page: "tutorials page"
+      contact_support: "contact Support"
+      hello: "Hello from the Archive of Our Own!"
+      html:
+        part1: "You're receiving this e-mail because you had works in a fanworks archive that has been imported by %{open_doors} into the %{ao3} (AO3). Because this e-mail address is connected to one registered on the imported archive, the associated fanworks (listed below) have been automatically added to your AO3 account."
+        part2: "You can read announcements about recent archive moves at %{ao3_news}, and find additional information on Open Doors' %{open_doors_faq} or %{tutorial_page}. For any questions not answered in the FAQ, tutorials, or this e-mail, please %{support}."
+        part3: "If this is a mistake and these are not your works, please don't delete them! Please %{open_doors} and we will sort it out."
+        part4: "Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please %{support}."
+        part5: "These works were written under the e-mail: %{email}"
+        part6: "To preserve rec lists and bookmarks, the imported archive's web addresses may redirect to the imported copy of these works for a limited time (check the announcement post for your archive to be sure). If you've already uploaded a copy of these works and you did NOT use the import from URL feature, there will be two copies of the same work on the archive."
+        part7: "If you would like Open Doors to update the redirect to point to your pre-existing work, please delete the imported copy, and %{open_doors} with your AO3 account name, your account name on the imported archive, and the title and URL of the fanwork you would like the redirect to point to. (If you have multiple works you would like to change the redirects for, you can list these in one email.)"
+        part8: "If you had other works on the imported archive under an e-mail address you can no longer access, please %{open_doors} with any information that can help verify your identity."
+        part9: "For other inquiries, please %{support}."
+        part10: |-
+               If you're contacting us, please whitelist email addresses from @transformativeworks.org and check your spam folders for our reply.
+        closing: |-
```

Comment:
Can we divide this into three separate bits like we did for `invitation_to_claim`? It's better for us to include the HTML in the email where possible.